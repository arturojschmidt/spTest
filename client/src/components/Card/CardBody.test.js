
import React from 'react';
import { shallow } from 'enzyme';
import CardBody from './CardBody.js';

describe('CardBody', () => {
 it('should be defined', () => {
   expect(CardBody).toBeDefined();
 });
 it('should render correctly', () => {
   const tree = shallow(
     <CardBody name='card body test' />
   );
   expect(tree).toMatchSnapshot();
 });
});