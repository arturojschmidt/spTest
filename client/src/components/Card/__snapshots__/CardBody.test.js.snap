// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBody should render correctly 1`] = `ShallowWrapper {}`;
